### PR TITLE
Abdul/helloblinky

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for HelloBlinky app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : Blinky.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : Blinky.cppproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
    PlatformToolSet from 141 to 142.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Changed the Board from Raspberry Pi to Upboard.
    Added the app *packages installing steps

### CODE REVIEW

File: Hello Blinky/CS/Blinky.csproj [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">ARM</Platform>
+    <ProjectGuid>{3662C7F2-04F8-4046-BF8E-EF1AC302DFE2}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Blinky</RootNamespace>
+    <AssemblyName>Blinky</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
```
Comment (neilsh-msft) [inline Hello Blinky/CS/Blinky.csproj]: These targetplatform ids were changed in other projects pull requests.  Why not this one?

File: Hello Blinky/Cpp/BlinkyCpp.vcxproj [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{61e2ec48-3271-4024-9bad-691cadf541af}</ProjectGuid>
+    <RootNamespace>BlinkyCpp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion>10.0.10240.0</WindowsTargetPlatformMinVersion>
```
Comment (neilsh-msft) [inline Hello Blinky/Cpp/BlinkyCpp.vcxproj]: These target platform version numbers were changed in other pulll requests for other projects.  Why not this one?

File: Hello Blinky/Cpp/BlinkyCpp.vcxproj [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
+    <ConfigurationType>Application</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <PlatformToolset>v141</PlatformToolset>
```
Comment (neilsh-msft) [inline Hello Blinky/Cpp/BlinkyCpp.vcxproj]: Other projects had their PlatformToolset changed to 142.  Why not this one?

File: Hello Blinky/CS/Blinky.csproj (hunk 1 above)
Comment (v-abdrahman) [reply Hello Blinky/CS/Blinky.csproj]: 10.0.19041.0

Changed the TargetPlatformVersion.

File: Hello Blinky/Cpp/BlinkyCpp.vcxproj (hunk 2 above)
Comment (v-abdrahman) [reply Hello Blinky/Cpp/BlinkyCpp.vcxproj]: 10.0.19041.0

Changed the TargetPlatformVersion.

File: Hello Blinky/Cpp/BlinkyCpp.vcxproj (hunk 3 above)
Comment (v-abdrahman) [reply Hello Blinky/Cpp/BlinkyCpp.vcxproj]: Changed the PlatformToolSet to v142

Changed the TargetPlatformVersion.

**Review by rsameser — APPROVED** [review]

**Pull request merged**